git remote remove origin--- a/data/battle_anim_scripts.s
+++ b/data/battle_anim_scripts.s
@@ -38007,6 +38007,108 @@
 @ routed to AnimTask_ThePitAseNoOp so no raw FireRed pointers remain.
 @ -----------------------------------------------------------------------------
 
+@ Export The Pit .ase scripts so C move data can link against them.
+	.global gBattleAnimMove_ThePit_AbyssMiasma
+	.global gBattleAnimMove_ThePit_AbyssWrath
+	.global gBattleAnimMove_ThePit_AdNihilo
+	.global gBattleAnimMove_ThePit_AdNihilo_Sub_1D3F7E
+	.global gBattleAnimMove_ThePit_AdNihilo_Sub_1D4138
+	.global gBattleAnimMove_ThePit_AkashicNova
+	.global gBattleAnimMove_ThePit_ArcaneBlast
+	.global gBattleAnimMove_ThePit_AstralGleam
+	.global gBattleAnimMove_ThePit_AtrophicEye
+	.global gBattleAnimMove_ThePit_AzureFlood
+	.global gBattleAnimMove_ThePit_Bafflingstep
+	.global gBattleAnimMove_ThePit_BrightSong
+	.global gBattleAnimMove_ThePit_CallAllies
+	.global gBattleAnimMove_ThePit_CallLion
+	.global gBattleAnimMove_ThePit_CallOwl
+	.global gBattleAnimMove_ThePit_CallSnake
+	.global gBattleAnimMove_ThePit_CellShield
+	.global gBattleAnimMove_ThePit_Chainweather
+	.global gBattleAnimMove_ThePit_ChainKiller
+	.global gBattleAnimMove_ThePit_Corkscrew
+	.global gBattleAnimMove_ThePit_CosmicSlash
+	.global gBattleAnimMove_ThePit_CrazedShout
+	.global gBattleAnimMove_ThePit_Darknessfist
+	.global gBattleAnimMove_ThePit_DemonicRush
+	.global gBattleAnimMove_ThePit_DivineStorm
+	.global gBattleAnimMove_ThePit_Doublehammer
+	.global gBattleAnimMove_ThePit_Electroshock
+	.global gBattleAnimMove_ThePit_EnergyEdge
+	.global gBattleAnimMove_ThePit_EthericBoon
+	.global gBattleAnimMove_ThePit_FirstPress
+	.global gBattleAnimMove_ThePit_FlareMask
+	.global gBattleAnimMove_ThePit_Freezeripple
+	.global gBattleAnimMove_ThePit_FreezeMask
+	.global gBattleAnimMove_ThePit_FrostCurse
+	.global gBattleAnimMove_ThePit_Fukubari
+	.global gBattleAnimMove_ThePit_Harvestparty
+	.global gBattleAnimMove_ThePit_Heal
+	.global gBattleAnimMove_ThePit_IcyAria
+	.global gBattleAnimMove_ThePit_IronDrill
+	.global gBattleAnimMove_ThePit_Izuna
+	.global gBattleAnimMove_ThePit_Kumogakure
+	.global gBattleAnimMove_ThePit_LeadingBlow
+	.global gBattleAnimMove_ThePit_LifeBallad
+	.global gBattleAnimMove_ThePit_LifeControl
+	.global gBattleAnimMove_ThePit_LightShot
+	.global gBattleAnimMove_ThePit_LimitBreak
+	.global gBattleAnimMove_ThePit_Lullaby
+	.global gBattleAnimMove_ThePit_Megalodive
+	.global gBattleAnimMove_ThePit_Mesmerize
+	.global gBattleAnimMove_ThePit_Meteor
+	.global gBattleAnimMove_ThePit_MindBash
+	.global gBattleAnimMove_ThePit_NobleOrder
+	.global gBattleAnimMove_ThePit_OceanRave
+	.global gBattleAnimMove_ThePit_OneTwopunch
+	.global gBattleAnimMove_ThePit_OriginCrush
+	.global gBattleAnimMove_ThePit_PanicSpores
+	.global gBattleAnimMove_ThePit_Phalanxguard
+	.global gBattleAnimMove_ThePit_PotionBomb
+	.global gBattleAnimMove_ThePit_PrimalDrums
+	.global gBattleAnimMove_ThePit_RadiantStar
+	.global gBattleAnimMove_ThePit_RapidFire
+	.global gBattleAnimMove_ThePit_RockHammer
+	.global gBattleAnimMove_ThePit_RottenEgg
+	.global gBattleAnimMove_ThePit_RoyalOrder
+	.global gBattleAnimMove_ThePit_ScytheDance
+	.global gBattleAnimMove_ThePit_Sealinglight
+	.global gBattleAnimMove_ThePit_ShatterShot
+	.global gBattleAnimMove_ThePit_ShogunDecoy
+	.global gBattleAnimMove_ThePit_Slashrampage
+	.global gBattleAnimMove_ThePit_SolarFlame
+	.global gBattleAnimMove_ThePit_SonicRaid
+	.global gBattleAnimMove_ThePit_SoulTrade
+	.global gBattleAnimMove_ThePit_Sporebarrage
+	.global gBattleAnimMove_ThePit_SpringTango
+	.global gBattleAnimMove_ThePit_StarDrop
+	.global gBattleAnimMove_ThePit_StompCrush
+	.global gBattleAnimMove_ThePit_Strangeseeds
+	.global gBattleAnimMove_ThePit_SubzeroGale
+	.global gBattleAnimMove_ThePit_SummerTango
+	.global gBattleAnimMove_ThePit_TagenBattou
+	.global gBattleAnimMove_ThePit_TimeCircle
+	.global gBattleAnimMove_ThePit_TitanKiller
+	.global gBattleAnimMove_ThePit_ToxicHerb
+	.global gBattleAnimMove_ThePit_ToxicReap
+	.global gBattleAnimMove_ThePit_VeilOrder
+	.global gBattleAnimMove_ThePit_VenomGust
+	.global gBattleAnimMove_ThePit_VoltMask
+	.global gBattleAnimMove_ThePit_VoltWings
+	.global gBattleAnimMove_ThePit_Wakingchakra
+	.global gBattleAnimMove_ThePit_WardingMist
+	.global gBattleAnimMove_ThePit_WaterBomb
+	.global gBattleAnimMove_ThePit_WaterTackle
+	.global gBattleAnimMove_ThePit_WeirdPotion
+	.global gBattleAnimMove_ThePit_WhipRipper
+	.global gBattleAnimMove_ThePit_WiltMiasma
+	.global gBattleAnimMove_ThePit_WingTalon
+	.global gBattleAnimMove_ThePit_Yggdrain
+	.global gBattleAnimMove_ThePit_Yggdraslash
+	.global gBattleAnimMove_ThePit_5RingSword
+	.global ThePitAseReturn
+
 	.align 2
 gBattleAnimMove_ThePit_AbyssMiasma:
 	@ Ported from FireRed The Pit export move_144_abyss_miasma.ase.txt.
